### DISCUSSION TITLE
ws2812 debugging

### DISCUSSION THREAD

**Discussion by ivynetca** [jomjol/AI-on-the-edge-device#1054] [Q&A] [answered]
Just installed a ws2812 led strip, but I am not seeing it being driven. I have put a scope on GPIO12, but dont see any activity.

Any suggestions on how to debug? I have followed config as outlined in wiki.

Thanks - Jeff

**Comment by jomjol** [comment]
Hard to say. If you connected according to description and have the correct power supply. Did you check the settings in config.ini?

**Reply by ivynetca** [reply]
GPIO settings in config.ini are as follows:
[GPIO]
;IO0 = input disabled  false false 
;IO1 = input disabled  false false 
;IO3 = input disabled  false false 
IO4 = built-in-led disabled  false false 
IO12 = external-flash-ws281x disabled  false false 
;IO13 = output disabled  false false 
LEDType = WS2812B
LEDNumbers = 2
LEDColor = 253 253 253

Not sure what the  "disabled  false false" are after the "IO12 = external-flash-ws281x" line
There isn't anything on the config screen to change this that I can see. Can/should I directly edit the config.ini to change something?

**Comment by ivynetca** [comment, marked as answer]
I reloaded the SD card from the latest release, and now working.
